dulwich/ssh_keygen.py

```diff
@@ -0,0 +1,296 @@
+# (c) 2018 Mantas Mikulėnas <grawity@gmail.com>
+# (c) 2024 E. Castedo Ellerman <castedo@castedo.com>
+# Released under the MIT License (https://spdx.org/licenses/MIT)
```

Comment:
Please use the standard preamble (Apache-2/GPLv2+) as elsewhere in the project, or include the MIT license here verbatim since we don't explicitly ship a copy anywhere.

Reply:
I like the option of including the MIT license. Short and less viral! :+1: Will do that.

Reply:
I just educated myself on the Apache license and now understand it is not copyleft. So I'll just use the standard preamble to keep things simple. My desire is to be contribute to the public domain and avoid copylefting unless really needed.